### ISSUE TITLE
add styleBase option to documentation

### ISSUE THREAD

**Issue by ddelella** [snapappointments/bootstrap-select#2125]
A suggestion to make the default styling match more closely to that of a form input field instead of a button.  In the current markup, a button element is being added directly after the select element.  This button element has the class **btn** and **btn-light**.  I tried overriding the style with form-control but the override only replaced the **btn-light**.  The **btn** class still remained.  To make this happen currently you would need the following JavaScript:

`$('.selectpicker').selectpicker({
    style: 'form-control'
});
$('.selectpicker + button').removeClass('btn');`

I suggest we alter the style property to replace both the **btn** and **btn-light** classes or we just make form-control the default class on the button element.  I have attached a screen shot of the two side by side and the code change.

![image](https://user-images.githubusercontent.com/548747/47310307-ad451880-d604-11e8-8237-581cd1c97b1f.png)

**Comment by caseyjhol** [comment]
You can use the `styleBase` option (which by default is set to `'btn'`). For some reason this isn't in the documentation.

**Comment by abrambailey** [comment]
Figured it out 
```
    $('select').selectpicker({
      liveSearch: true,
      style: '',
      styleBase: 'form-control'
    });
```

**Comment by caseyjhol** [comment]
Released in [v1.13.4](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.4)!
